scripts/dockerrun.sh

```diff
@@ -115,7 +115,7 @@ execute() {
 echo "Building Docker Container using Dockerfile: $DOCKERFILE"
 
 # Get the name of Docker image.
-image=$(grep -i "^FROM " "$DOCKERFILE" | awk '{ print $2 }')
+image=$(grep -i "^FROM " "$DOCKERFILE/Dockerfile" | awk '{ print $2 }')
```

Comment:
I think we probably want to support both scenarios: `DOCKERFILE` being a path to the dir, as well as a direct path to the Dockerfile. As far as I can tell, the rest of the scripts work fine with either, and I don't think we should enforce the `Dockerfile` name convention.

I think a solution would be to check for `$DOCKERFILE` being a dir, and if it is, try `$DOCKERFILE/Dockerfile`.